bindings/python/visualize/meshcat_visualizer.py

```diff
@@ -633,7 +633,12 @@ def load_model(self, model: idyn.Model, model_name="iDynTree", color=None):
         self.traversal[model_name] = idyn.Traversal()
         self.link_pos[model_name] = idyn.LinkPositions()
 
-        self.model[model_name].computeFullTreeTraversal(self.traversal[model_name])
+        if base_frame is None:
```

Comment:
```suggestion
        if base_link is None:
```

I guess this should also change?